README.md

```diff
@@ -108,3 +102,151 @@ with Client("//127.0.0.1", exampleStub) as client:
         print(response.message)
 
 ```
+
+### Protobuf
+
+The protobuf for the above examples is:
+
+```
+syntax = "proto3";
+
+package nameko;
```

Comment:
Is this required? According to this doc: https://developers.google.com/protocol-buffers/docs/proto#packages 
```
In Python, the package directive is ignored, since Python modules are organized according to their location in the file system.
```

Reply:
I'm a bit confused by this note in the doc. That package name definitely shows up in things like HTTP headers.

Reply:
As grpc is about providing cross language support to Nameko adding package name to proto definition should be a best practice anyway.